ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -189,6 +189,14 @@ objects:
         done
         # oh god the blood
         sleep 180
+        # temporary - remove this check once a proper test is added to origin e2e
+        password=$(cat /tmp/artifacts/installer/auth/kubeadmin-password)
+        sysadmin=$(oc config --config=${KUBECONFIG} current-context)
+        oc config --config=$KUBECONFIG set-context "${sysadmin}"
+        if ! oc --config=$KUBECONFIG login -u kubeadmin -p "${password}"; then
+          echo "failed to login as kubeadmin" && exit 1
```

Comment:
I don't see what value failing has here?

Reply:
All the other steps are "waits", not fails

Reply:
done, made it an `until..; do \ sleep 10`